packages/orbit-components/src/Icon/IconList.tsx

```diff
@@ -1,62 +1,36 @@
 import * as React from "react";
-import styled from "styled-components";
-import { defaultTokens } from "@kiwicom/orbit-design-tokens";
 
 import * as Icons from "../icons";
 
-const List = styled.div`
-  display: flex;
-  flex-flow: row wrap;
-  justify-content: space-between;
-  border-radius: ${defaultTokens.borderRadiusLarge};
-`;
-
-const Container = styled.div`
-  display: flex;
-  flex-direction: row;
-  gap: ${defaultTokens.spaceLarge};
-  justify-content: flex-start;
-  align-items: center;
-  align-content: center;
-  width: 100%;
-  min-height: 80px;
-  background-color: white;
-  margin-bottom: ${defaultTokens.spaceLarge};
-  border-radius: ${defaultTokens.borderRadiusLarge};
-  border: ${defaultTokens.borderWidthCard} ${defaultTokens.borderStyleCard}
-    ${defaultTokens.paletteCloudNormal};
-  padding: 0 ${defaultTokens.spaceLarge};
-`;
-
-const IconImport = styled.div`
-  font-family: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, Courier, monospace;
-  font-size: 12px;
-  line-height: ${defaultTokens.lineHeightTextSmall};
-  color: ${defaultTokens.paletteInkDark};
-  background-color: ${defaultTokens.paletteCloudLight};
-  border: ${defaultTokens.borderWidthCard} ${defaultTokens.borderStyleCard}
-    ${defaultTokens.paletteCloudNormal};
-  padding: ${defaultTokens.spaceXXSmall} ${defaultTokens.spaceXSmall};
-`;
-
 const IconList = () => (
-  <List>
+  <div className="rounded-large flex flex-row flex-wrap justify-between">
     {Object.keys(Icons)
       .filter(n => n !== "__namedExportsOrder")
       .map(icon => {
         const Icon = Icons[icon];
 
         const iconName = `${icon}`;
         return (
-          <Container key={icon}>
+          <div
+            className="gap-lg bg-white-normal mb-lg rounded-large border-cloud-normal px-lg flex w-full flex-row content-center items-center justify-start border border-solid"
```

Comment:
We don't have specific classes for `borderWidthCard` and `borderStyleCard` tokens, right?

Also, is the class order somehow automatic? It'd be nice to have all the border related things in the same area.

One idea: in CubeCSS methodology, they separate concerns with a `|` character (and of course, the `|` class does nothing)

Reply:
Those tokens are deprecated, since they've proved to be useless. The width is just 1px (class `border`) and the style is just "solid" (class `border-solid`). So we don't need them.

We have Prettier for the order, so the only way to actually control the order is if we use the `cx` function and separate into multiple strings… But we haven't been paying much attention to that, to be honest